src/lib/daemon_rpcs/daemon_rpcs.ml

```diff
@@ -516,6 +516,17 @@ module Snark_job_list = struct
     Rpc.Rpc.create ~name:"Snark_job_list" ~version:0 ~bin_query ~bin_response
 end
 
+module Snark_pool_list = struct
+  type query = unit [@@deriving bin_io]
+
+  type response = string [@@deriving bin_io]
```

Comment:
I would have preferred exposing a `Yojson.Safe.t` (which probably doesn't have bin_prot stuff...), but a string here seems fine for the intended use.